### ISSUE TITLE
Create custom knob component

### ISSUE THREAD

**Issue by bcrussin** [bcrussin/web-synth#5]
To match many VSTs and synthesizer plugins, most range inputs should be replaced with a custom knob component. This component should follow web accessibility guidelines (including keyboard support).
